tests/ui_tests/test_service_page.py

```diff
@@ -289,7 +293,7 @@ def test_save_advanced_config_on_service_config_page(self, app_fs, sdk_client_fs
         service_config_page.config.click_on_advanced()
         with allure.step("Check that rows are visible"):
             config_rows = service_config_page.config.get_all_config_rows()
-            assert len(config_rows) == 16, "Rows should be visible"
+            assert len(config_rows) == params["rows_amount"], f"There are should be {'rows_amount'} rows"
```

Comment:
`"There should be {params['rows_amount']} rows"`

Reply:
thnx, fixed

Reply:
thnx, fixed

Reply:
thnx, fixed